src/AppBundle/Entity/Location.php

```diff
@@ -216,4 +238,38 @@ public function getAddress()
     {
         return $this->address;
     }
+
+    /**
+     * Add voteSetting
+     *
+     * @param \AppBundle\Entity\VoteSettings $voteSetting
+     *
+     * @return Location
+     */
+    public function addVoteSetting(\AppBundle\Entity\VoteSettings $voteSetting)
+    {
+        $this->voteSetting[] = $voteSetting;
```

Comment:
why add remove? If needed - create function saveVoteSetting and if you want clear first all collection element you dont need neccessary every element collection, ArrayCollection have function "clear" but if oyu nedded in saveVoteSetting yoo can call remove to/ But in addVoteSetiing dont nedded call remove